lib/src/navigation_state.dart

```diff
@@ -1,14 +1,14 @@
+import 'package:flutter_redux_navigation/src/navigation_destination.dart';
+
 /// It keeps the current and previous path of the navigation.
 class NavigationState {
-  final String previousPath;
-  final Object previousArguments;
-  final String currentPath;
```

Comment:
Please keep the `currentPath` and `previousPath` as deprecated setters that will internally initialize a `NavigationDestination` without `arguments`, this way we will avoid introducing breaking changes and gradually remove this functionality.

Reply:
I haven't used the `@Deprecated` keyword before, but seems pretty straight forward, so I submitted that fix now.